README.md

```diff
@@ -61,6 +63,35 @@ in the root of the `meteor` repository. The typical symptom of not
 updating submodules will be `Error: Depending on unknown package ...`
 when you run most Meteor commands.
 
+### Create testing app
```

Comment:
Agree that the user knows how to create a Meteor app. Otherwise you don't need a local checkout.

As far as I can see though it's important that the app gets created by the custom installed version (which is why referring to ./meteor/meteor instead of just meteor create) which is consistent with the part on how to run the tests as described here: https://github.com/meteor/meteor/blob/devel/Contributing.md#run-against-your-local-meteor-copy. 

Very open to improvements though to make things even more clear.